geoana/em/static.py

```diff
@@ -70,10 +69,13 @@ def magnetic_flux(self, xyz, **kwargs):
 
     @staticmethod
     def magnetic_flux_equation():
-        return Latex(
+        return "{}".format(
```

Comment:
Idea was for this to show up in the notebook nicely.

Reply:
yeah, I think `Latex` was removed from ipywidgets. I can no longer import it...